app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,3 @@
+class PostsController < ApplicationController
+  def index; end
+end
```

Comment:
- [ ] [REQUIRED] Good job so far! The `posts_controller` shoud have `#index` ✔️ and `#show`❌ actions. You missed add `#show`, please add it 🙏🏼